client/src/components/MultiJurisdictionAudit/Progress/index.test.tsx

```diff
@@ -13,7 +13,7 @@ describe('Progress screen', () => {
     const { container } = render(
       <Progress
         jurisdictions={jurisdictionMocks.oneManifest}
-        auditSettings={auditSettings.all}
+        auditSettings={auditSettings.all!}
```

Comment:
do you still need all these exclamation points?

Reply:
@jonahkagan Evidently not, haha. Fixed and pushed. :slightly_smiling_face: